src/Microsoft.OData.Client/Binding/BindingObserver.cs

```diff
@@ -381,8 +381,8 @@ internal void OnDataServiceCollectionChanged(object collection, NotifyCollection
                     break;
 #endif
 
-                default:
-                    throw new InvalidOperationException(Strings.DataBinding_DataServiceCollectionChangedUnknownActionCollection(eventArgs.Action));
+                default: // No Action

```

Comment:
It seems it is a breaking change. Could we consider it:
1) have a throw exception config
2) If no config or  throw exception config, throw as previous
3) if config to non-exception, no exception throw
?

Reply:
I would rather not add config for things like this as they just clutter code and confuse customers. In this case, going from an error state to an ignore state in the unlikely event that .NET extends the enum seems a pretty safe change.

---
In reply to: [355600478](https://github.com/OData/odata.net/pull/1601#discussion_r355600478) [](ancestors = 355600478)